messages/create.sandbox.md

```diff
@@ -123,3 +123,19 @@ The poll interval (%d seconds) can't be larger that wait (%d in seconds)
 # error.noCloneSource
 
 Could not find the clone sandbox name "%s" in the target org.
+
+# error.userGroupQueryFailed
+
+Could not query user group.
```

Comment:
ditto https://github.com/salesforcecli/plugin-org/pull/1221/files#r1773460127

Reply:
I like Cristian's suggestion below, so how about this:

`Unable to find the ID of the activation user group <activationGroupName> that you defined in the definition file. `
I assume that in your code, <activationGroupName> will actually be something like `<%s>`, right?